middleman/chat_functions.py

```diff
@@ -88,7 +90,16 @@ async def send_text_to_room(
         logger.exception(f"Unable to send message response to {room_id}")
         return f"Failed to send message: {ex}"
 
-
+async def get_dm(client: AsyncClient, user: str): 
+    room_id = self.store.get_dm(user_id=user)
+    if room_id == "0":
+        resp = client.room_create(
+            is_direct=True,
+            invite=[user]
+        )
+        room_id = resp.room_id
+        self.store.store_dm(user_id=user, room_id=room_id)
+    return room_id
 async def send_reaction(
```

Comment:
```suggestion
    return room_id


async def send_reaction(
```

Functions should be separated by two blank lines, see https://pep8.org/#blank-lines